### PR TITLE
add support for config generation

### PR DESCRIPTION
**Pull request by mceloud**

### CODE REVIEW

File: rabbitmq/meta/config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+config:
+  {%- if pillar.rabbitmq.server is defined %}
+  {%- from "rabbitmq/map.jinja" import server with context %}
```
Comment (fpytloun) [inline rabbitmq/meta/config.yml:3]: This is not needed

Comment (mceloud) [reply rabbitmq/meta/config.yml:3]: Using the same approach like: https://github.com/tcpcloud/salt-formula-galera/commit/4bca6a38e1838f504946ba53eb9e31010b4b39b8. Trying to keep it same.

**Comment by pupapaik** [comment]
Does this work or not?

**Comment by cznewt** [comment]
LGTM

**Pull request merged**